bookstore/_version.py

```diff
@@ -52,34 +51,38 @@ class NotThisMethod(Exception):
     """Exception raised if a method is not valid for the current scenario."""
 
 
-LONG_VERSION_PY = {}
-HANDLERS = {}
+LONG_VERSION_PY: dict = {}
+HANDLERS: dict = {}
```

Comment:
Introduced annotations here to make `mypy` happy (yup, kicking off type annotations).